samples/sample_fei/src/sample_fei.cpp

```diff
@@ -1454,7 +1454,7 @@ int main(int argc, char *argv[])
     msdk_printf(MSDK_STRING("\nProcessing finished after %.2f sec \n"), MSDK_GET_TIME(msdk_time_get_tick(), startTime, frequency));
     pipe_threads.clear();
 
-    for (auto & pipeline : pipelines)
+    for(std::vector<std::unique_ptr<CEncodingPipeline>>::reverse_iterator pPipeline = pipelines.rbegin(); pPipeline != pipelines.rend(); pPipeline++)
```

Comment:
you can use "auto pPipeline" to simplify things. But maybe iPipeline is better, since it is not a pointer